static/app/utils/useOverlay.tsx

```diff
@@ -183,10 +158,43 @@ function useOverlay({
     ],
     [arrowElement, offset, preventOverflowOptions]
   );
-  const {styles: popperStyles, state: popperState} = usePopper(
-    triggerElement,
-    overlayElement,
-    {modifiers, placement: position}
+  const {
+    styles: popperStyles,
+    state: popperState,
+    update: popperUpdate,
+  } = usePopper(triggerElement, overlayElement, {modifiers, placement: position});
+
+  // Get props for trigger button
+  const openState = useOverlayTriggerState({isOpen, defaultOpen, onOpenChange});
+  const {buttonProps} = useButton(
+    {
+      onPress: () => {
+        openState.open();
+        popperUpdate?.();
+      },
+    },
```

Comment:
@evanpurkhiser Added a call to `popperUpdate()` here. This is the only thing that's changed.

Reply:
🆒

Reply:
Actually just moved `popperUpdate()` to `useOverlayTriggerState()` (https://github.com/getsentry/sentry/pull/42824/commits/98afe417d004041282e031374405c6cd60654c88) — same effect, just making sure that the function is fired every time the open state changes and not just when the trigger button is clicked.